Inc/ST-LIB_HIGH/Protections/Boundary.hpp

```diff
@@ -57,57 +65,88 @@ template<class Type, ProtectionType Protector> struct Boundary;
 template<class Type>
 struct Boundary<Type, BELOW> : public BoundaryInterface{
 	static constexpr ProtectionType Protector = BELOW;
+	bool has_warning_level{false};
```

Comment:
why do you initilize it this way?

Reply:
i could do `bool has_warning_level = false;`but it just personal preference